### PR TITLE
#163384443 adjust UI pages style 

### PR DESCRIPTION
**Pull request by Jesse-efe**
#### What does this PR do?
Make adjustments to UI pages
#### Description of Task to be completed?
Add nav bar to sign-in, sign-up, and reset-password pages
Remove empty space above footer in homepage
Create a new page that shows all parties
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#163384443
#### Screenshots
one
two
three
five
four

#### Questions:
N/A

### CODE REVIEW

**Comment by megame24** [comment]
Kindly follow the instructions on this Github page(https://github.com/andela/workdey/wiki/PR-Format) to format your PR. Reply with `N/A` to PR description question(s) that does not apply to this PR.
Also, on the same page, take a look at the right to see the commit message convention, incorporate the convention and update your commit message.

**Comment by megame24** [comment]
Since this task has to do with the UI, please include screenshots to the PR description.

**Comment by Jesse-efe** [comment]
Thank you @megame24. I have implemented the changes

**Comment by megame24** [comment]
Let's find another color combination for the yellow sign up button, I can barely see the text in the button.

**Comment by Jesse-efe** [comment]
Ok Let me see

**Comment by megame24** [comment]
Your create account form has a black button labeled `sign in`, let's keep the color of the buttons consistent, also, update the name on the button to reflect the appropriate form.

**Comment by Jesse-efe** [comment]
Ok I will do so now

File: UI/styles/parties.css [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+}
+/* @media (max-width: 790px) {
+    #container > .summary{
+        grid-template-columns: repeat(2, 1fr);
+        grid-gap: 2rem;
+    }
+    #container > .description{
+        grid-template-columns: 1fr;
+    }
+}  */
```
Comment (megame24) [inline UI/styles/parties.css]: Get rid of commented block of code

**Review by megame24 — APPROVED** [review]
Implement the last feedback and merge. Good job.

File: UI/styles/parties.css (hunk 1 above)
Comment (Jesse-efe) [reply UI/styles/parties.css]: Ok

**Comment by megame24** [comment]
Let's maintain a consistent color for both the primary button(blue) and the secondary button(yellow). What do think about changing the green buttons to the blue ones(primary button)?

**Comment by Jesse-efe** [comment]
Ok I would try that

**Comment by megame24** [comment]
This is ready for merge, good job

**Comment by Jesse-efe** [comment]
Thank you

**Pull request merged**